src/chrome/content/rules/Fedorainfracloud.org.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+        Note that the Fedora Infrastructure Cloud is used for various
+        community-lead projects which are not directly supported by Fedora
+        Infrastructure. Therefore, we cannot wildcard here, and must explicitly
+        list those fedorainfracloud.org subdomains which have a valid SSL
+        certificate.
+-->
+<ruleset name="Fedora Infrastructure Cloud (partial)">
+	<target host="copr.fedorainfracloud.org" />
+	<!--<target host="grafana.fedorainfracloud.org" />-->
+	<target host="graphite.fedorainfracloud.org" />
+	<target host="piwik.fedorainfracloud.org" />
+	<target host="taiga.fedorainfracloud.org" />
```

Comment:
There are also:
fedorainfracloud.org
www.fedorainfracloud.org
developer.fedorainfracloud.org (refused)
faitout.fedorainfracloud.org (timeout)
eclipse.fedorainfracloud.org (currently down?)

So this rule don't need to be partial.